pkgs/development/python-modules/netbox-floorplan-plugin/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  netbox,
+}:
+buildPythonPackage rec {
+  pname = "netbox-floorplan-plugin";
+  version = "0.6.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "netbox-community";
+    repo = "netbox-floorplan-plugin";
+    rev = version;
+    hash = "sha256-cJrqSXRCBedZh/pIozz/bHyhQosTy8cFYyji3KJva9Q=";
+  };
+
+  build-system = [ setuptools ];
+
+  nativeCheckInputs = [ netbox ];
+
+  preFixup = ''
+    export PYTHONPATH=${netbox}/opt/netbox/netbox:$PYTHONPATH
+  '';
+
+  pythonImportsCheck = [ "netbox_floorplan" ];
+
+  meta = with lib; {
+    description = "A netbox plugin providing floorplan mapping capability for locations and sites.";
```

Comment:
```suggestion
    description = "Netbox plugin providing floorplan mapping capability for locations and sites";
```

Reply:
I'm not too sure what is the right choice here, @felbinger did previously suggest the `.` at the end. Is there some sort of guideline for descriptions?

Reply:
Nevermind I was wrong ;)

Reply:
The contributing guide contains a word about meta.description and there where recently some treewide PRs to fix those.

Reply:
Thank you for the comment